api/v1alpha4/condition_consts.go

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Now that v1beta1 is created, add api/v1beta1/condition_consts.go instead of this,please.